assets/queries/ansible/gcp/log_temp_files_wrong_value/query.rego

```diff
@@ -0,0 +1,37 @@
+package Cx
+
+CxPolicy [ result ] {
+  document := input.document[i]
+  tasks := getTasks(document)
+  task := tasks[t]
+  instance := task["google.cloud.gcp_sql_instance"]
+  settings := instance.settings
+  database_flags := settings.database_flags
+
+  check_database_flags_content(database_flags)
+
+  result := {
+                "documentId": 		document.id,
+                "searchKey": 	    sprintf("name=%s.{{google.cloud.gcp_sql_instance}}.settings.database_flags", [task.name]),
+                "issueType":		"IncorrectValue",
+                "keyExpectedValue": sprintf("name=%s.{{google.cloud.gcp_sql_instance}}.settings.database_flags sets the log_temp_files to 0", [task.name]),
+                "keyActualValue":   sprintf("name=%s.{{google.cloud.gcp_sql_instance}}.settings.database_flags doesn't set the log_temp_files to 0", [task.name])
+              }
+}
```

Comment:
Is it a vulnerability if 'log_temp_files' is not set?  'The default setting is -1, which disables such logging.'

Reply:
I haven't seen any example that 'log_temp_files' has a default value and we are only checking if the 'log_temp_files' value is 0.